execution/rerun/rerun.go

```diff
@@ -37,32 +37,27 @@ import (
 )
 
 const (
-	dotGauge     = ".gauge"
-	infoFileName = "runInfo.json"
+	dotGauge   = ".gauge"
```

Comment:
There is already a constant for .gauge folder defined in common. We should be using that variable everywhere.